pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -202,28 +203,34 @@ func (r *ReconcileComputeNodeOpenStack) Reconcile(request reconcile.Request) (re
 		//    - run cleanup jobs to remove compute node service entries in OSP control plane
 		log.Info(fmt.Sprintf("CR %s delete, running cleanup", instance.Name))
 		err := deleteAllBlockerPodFinalizer(r, instance)
-		if err != nil {
+		if err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
seems like we should do the ifs in the opposite order to not have to repeat the errors.IsAlreadyExists in both if clauses

Reply:
done